### PR TITLE
Create .deepsource.toml

### PR DESCRIPTION
**Pull request by QtKaii**
Allow deepsource to analyse the repository

### CODE REVIEW

File: .deepsource.toml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version = 1
+
+[[analyzers]]
+name = "shell"
+enabled = true
+
+[[analyzers]]
+name = "javascript"
```
Comment (Tolfx) [inline .deepsource.toml:8]: Isn't there a typescript support? Or maybe supports typescript too?

Comment (QtKaii) [reply .deepsource.toml:8]: Javascript includes typescript support
[Reference](https://deepsource.io/blog/release-js-beta/)

Comment (Tolfx) [reply .deepsource.toml:8]: Looks Good

**Review by Tolfx — APPROVED** [review]
LGTM

**Pull request merged**